lib/forest_liana/bootstraper.rb

```diff
@@ -100,6 +101,19 @@ def check_integrations_setup
       end
     end
 
+    def namespace_duplicated_models
+      ForestLiana.models
+        .group_by {|model| model.table_name}
+        .select {|table_name, models| models.length > 1}
+        .try(:each) do |table_name, models|
+          models.each do |model|
+            ForestLiana.names_overriden[model] =
+              "#{model.name.deconstantize.downcase}__#{model.table_name}"
```

Comment:
Here's the fix for the belongs_to issue you've found. We convert the model name to downcase